src/lib/core/style/_checkbox-variables.scss

```diff
@@ -0,0 +1,10 @@
+@import './variables';
```

Comment:
File should be `_checkbox-common.scss` to be consistent with other shared styles